test/iterator.js

```diff
@@ -610,6 +610,20 @@ metatests.testSync("Iterator.toObject with '0', '1' properties", test => {
   test.strictSame(actual, { a: 0, b: 1, c: 2 });
 });
 
+metatests.testSync('Iterator.chunks', test => {
```

Comment:
Please add tests:
* empty array
* condition that will result in one huge chunk
* `[1, 1, 3, 3, 3, 5, 5].chunks(x => x === 3)`